src/modules/playRoom/components/PickItem.js

```diff
@@ -1,29 +1,116 @@
-import { useState } from 'react';
-import { Modal, Input } from 'antd';
+import { useEffect, useState } from 'react';
+import { Modal, Input, Form, notification } from 'antd';
+import PlayRoomAPIClient from '../services/PlayRoomAPIClient';
+import { socket } from '../../../core/services/socket';
+import store from '../../../core/store/index';
+import convertChoice from '../../../core/helper/convertChoice';
+import { NOTIFICATION_SUCCESS, BET_SUCCESS } from '../../../core/constants/successMessage';
+import { EMPTY_INPUT_MONEY_ERROR, EXCEED_MONEY_ERROR, NOTIFICATION_ERROR, PLAYER_CHOICE_ERROR } from '../../../core/constants/errorMessage';
 
-const PickItem = ({ img }) => {
+const PickItem = ({ img, isProcessingResult, setIsProcessingResult }) => {
+  const { dispatch } = store;
```

Comment:
Use mapDispatch

Reply:
i tried to use mapState and mapDispatch but when passing props to component it doesn't work